fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/LocalResourceReferenceRec.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.dao.impl;
+
+
+/**
+ * A DTO representing an external resource reference. This object is initialized
+ * with names which are resolved to ids when passed through the cache. The idea
+ * is to reduce the number of times we have to iterate over the list of records
+ */
+public class LocalResourceReferenceRec extends ResourceRefRec {
+
+    // The loca ref resource type
```

Comment:
```suggestion
    // The local ref resource type
```

Reply:
Will probably be removed in subsequent cleanup